Add hint for public dir to hosting config (#1050)
contao/docs: docs/manual/installation/system-requirements.en.md

Therefore, a separate (sub)domain is required for each Contao installation.
{{% /notice %}}

> ({{< version-tag "4.13" >}} When your installation is still using the folder `/web` as public directory, please explicitly set it in the `composer.json`

Reviewer: ```suggestion
{{< version-tag "4.13" >}} If your installation is still using the folder `/web` as its public directory, explicitly set it in the `composer.json`
```

> of the project, to be prepared for future version of contao:

Reviewer: ```suggestion
of the project in order to be prepared for future version of contao:
```

Author: ```suggestion
of the project in order to be prepared for future versions of contao:
```

```json
{
  "extra": {
    "public-dir": "web"
  }
}
```

> see also: https://symfony.com/doc/current/configuration/override_dir_structure.html#override-the-public-directory

Reviewer: ```suggestion

see also: https://symfony.com/doc/current/configuration/override_dir_structure.html#override-the-public-directory
```
